include/glow/Base/Tensor.h

```diff
@@ -347,6 +348,27 @@ class Tensor final {
     return *this;
   }
 
+  /// Dump a textual representation of the Tensor into provided output stream.
+  void dump(llvm::raw_ostream &os) const;
+
+  /// Dump a textual representation of the Tensor into default output stream.
+  void dump() const;
+
+  /// Dump a textual representation of the Tensor to std::string.
+  std::string toString() const;
+
+  friend llvm::raw_ostream &operator<<(llvm::raw_ostream &os, const Tensor &t) {
+    t.dump(os);
+    return os;
+  }
+
+  friend llvm::raw_ostream &operator<<(llvm::raw_ostream &os,
+                                       const Tensor *const t) {
```

Comment:
Do we really need `const Tensor *const ` compared to `const Tensor *` we typically use everywhere? There are more places like this in your PR.

Reply:
> Do we really need `const Tensor *const ` compared to `const Tensor *` we typically use everywhere? There are more places like this in your PR.

OK. I think they are the same. I removed the redundant const.